dotcom-rendering/src/components/ShareButton.importable.tsx

```diff
@@ -189,6 +189,12 @@ export const ShareButton = ({
 	const isLiveBlogMeta =
 		format.design === ArticleDesign.LiveBlog && context === 'ArticleMeta';
 
+	const isLiveBlogBlockDesktop =
+		typeof window !== 'undefined' &&
+		format.design === ArticleDesign.LiveBlog &&
+		context === 'LiveBlock' &&
+		window.innerWidth > 980;
```

Comment:
If we're attempting to activate certain features based on breakpoint in JS then I suspect we'll want `window.matchMedia`[^1]. We have a custom hook to do this here, based on React's `useSyncExternalStore`[^2]:

https://github.com/guardian/dotcom-rendering/blob/d4fcbe0ce911a956df7cc93cc5c2629ccdd69b32/dotcom-rendering/src/lib/useMatchMedia.ts#L13-L22

and you can see an example of it in use here:

https://github.com/guardian/dotcom-rendering/blob/d4fcbe0ce911a956df7cc93cc5c2629ccdd69b32/dotcom-rendering/src/components/AdPortals.importable.tsx#L124-L126

One of the reasons for doing this is that it handles screen resizing. At the moment the code above will only check the window size once on startup and never again, even if it changes.

[^1]: https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia
[^2]: https://react.dev/reference/react/useSyncExternalStore